WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -464,37 +468,34 @@ private async Task CreateRoundsAsync(CancellationToken cancellationToken)
 				var largeSuggestion = MaxSuggestedAmountProvider.AbsoluteMaximumInput;
 
 				var roundWithoutThis = Rounds.Except(new[] { round });
-				RoundParameters parameters = RoundParameterFactory.CreateRoundParameter(feeRate, largeSuggestion);
+				RoundParameters parameters = RoundParameterFactory.CreateRoundParameter(feeRate, smallSuggestion);
+				var smallRound = TryMineRound(parameters, roundWithoutThis.ToArray());
+
 				Round? foundLargeRound = roundWithoutThis
 					.FirstOrDefault(x =>
-									x.Phase == Phase.InputRegistration
-									&& x is not BlameRound
-									&& !x.IsInputRegistrationEnded(Config.MaxInputCountByRound)
-									&& x.Parameters.MaxSuggestedAmount >= allInputs.Max()
-									&& x.InputRegistrationTimeFrame.Remaining > TimeSpan.FromSeconds(60));
-				var largeRound = foundLargeRound ?? TryMineRound(parameters, roundWithoutThis.ToArray());
-
-				if (largeRound is not null)
+						x.Phase == Phase.InputRegistration
+						&& x is not BlameRound
+						&& !x.IsInputRegistrationEnded(Config.MaxInputCountByRound)
+						&& x.Parameters.MaxSuggestedAmount >= allInputs.Max()
+						&& x.InputRegistrationTimeFrame.Remaining > TimeSpan.FromSeconds(60));
+				parameters = RoundParameterFactory.CreateRoundParameter(feeRate, largeSuggestion);
+				var largeRound = foundLargeRound ?? TryMineRound(parameters, roundWithoutThis.Concat(new[] { smallRound }).ToArray());
```

Comment:
Nullability warning here.

Also,I got this on RegTest which is tightly related:

```
2022-10-06 11:14:29.359 [39] ERROR      PeriodicRunner.ExecuteAsync (100)       System.NullReferenceException: Object reference not set to an instance of an object.
   at WalletWasabi.WabiSabi.Backend.Rounds.Arena.<TryMineRound>b__56_0(Round x) in WalletWasabi\WabiSabi\Backend\Rounds\Arena.cs:line 534
   at System.Linq.Enumerable.WhereListIterator`1.ToArray()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.OrderedEnumerable`1.GetEnumerator()+MoveNext()
   at System.Linq.Enumerable.SelectIPartitionIterator`2.MoveNext()
   at System.Collections.Immutable.ImmutableDictionary`2.AddRange(IEnumerable`1 items, MutationInput origin, KeyCollisionBehavior collisionBehavior)
   at System.Collections.Immutable.ImmutableDictionary`2.AddRange(IEnumerable`1 pairs, Boolean avoidToHashMap)
   at System.Collections.Immutable.ImmutableDictionary.ToImmutableDictionary[TSource,TKey,TValue](IEnumerable`1 source, Func`2 keySelector, Func`2 elementSelector)
   at WalletWasabi.WabiSabi.Backend.Rounds.Arena.TryMineRound(RoundParameters parameters, Round[] rounds) in WalletWasabi\WabiSabi\Backend\Rounds\Arena.cs:line 539
   at WalletWasabi.WabiSabi.Backend.Rounds.Arena.CreateRoundsAsync(CancellationToken cancellationToken) in WalletWasabi\WabiSabi\Backend\Rounds\Arena.cs:line 482
   at WalletWasabi.WabiSabi.Backend.Rounds.Arena.ActionAsync(CancellationToken cancel) in WalletWasabi\WabiSabi\Backend\Rounds\Arena.cs:line 86
   at WalletWasabi.Bases.PeriodicRunner.ExecuteAsync(CancellationToken stoppingToken) in WalletWasabi\Bases\PeriodicRunner.cs:line 76
```

If this exc occurs, we are in a (in)finite loop, where we're heavily trying to create new round, which is either too expensive to create
`Round (xy): First ordered round creation too expensive. Skipping...`

or we got the exception above and we retry.